### ISSUE TITLE
依赖版本更新

### ISSUE THREAD

**Issue by irvinezhao** [WeBankPartners/wecube-platform#543]
**描述您遇到的bug**  

![image](https://user-images.githubusercontent.com/17263271/68356714-04555700-014e-11ea-978e-52e14ef286f1.png)

**如何重现** 

**预期行为** 
```
<dependency>
  <groupId>com.fasterxml.jackson.core</groupId>
  <artifactId>jackson-databind</artifactId>
  <version>[2.9.10.1,)</version>
</dependency>
```
**截图**  

**附加**

**Comment by junchensz** [comment]
当用<version>[2.9.10.1,)</version>时，版本会解释成2.10.0.pr3, 当 wecube-core启动时会报错，
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/core/exc/InputCoercionException
        at com.fasterxml.jackson.datatype.jsr310.JavaTimeModule.<init>(JavaTimeModule.java:141) ~[jackson-datatype-jsr310-2.9.8.jar!/:2.9.8]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_112]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_112]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_112]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_112]
        at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:172) ~[spring-beans-5.1.5.RELEASE.jar!/:5.1.5.RELEASE]
        ... 93 common frames omitted
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.core.exc.InputCoercionException
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[na:1.8.0_112]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[na:1.8.0_112]
        at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:93) ~[wecube-core.jar:1.0-SNAPSHOT]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[na:1.8.0_112]
        ... 99 common frames omitted

**Comment by irvinezhao** [comment]
ISSUE Fixed in #562
